src/pynwb/file.py

```diff
@@ -102,6 +101,112 @@ def __init__(self, **kwargs):
             setattr(self, key, val)
 
 
+_add_electrode_docval = (
+    {'name': 'x', 'type': 'float', 'doc': 'the x coordinate of the position (+x is posterior)',
+        'default': None},
+    {'name': 'y', 'type': 'float', 'doc': 'the y coordinate of the position (+y is inferior)', 'default': None},
+    {'name': 'z', 'type': 'float', 'doc': 'the z coordinate of the position (+z is right)', 'default': None},
+    {'name': 'imp', 'type': 'float', 'doc': 'the impedance of the electrode, in ohms', 'default': None},
+    {'name': 'location', 'type': str,
+     'doc': 'the location of electrode within the subject e.g. brain region. Required.',
+     'default': None},
+    {'name': 'filtering', 'type': str,
+     'doc': 'description of hardware filtering, including the filter name and frequency cutoffs',
+     'default': None},
+    {'name': 'group', 'type': ElectrodeGroup,
+     'doc': 'the ElectrodeGroup object to add to this NWBFile. Required.',
+     'default': None},
+    {'name': 'id', 'type': int, 'doc': 'a unique identifier for the electrode', 'default': None},
+    {'name': 'rel_x', 'type': 'float', 'doc': 'the x coordinate within the electrode group', 'default': None},
+    {'name': 'rel_y', 'type': 'float', 'doc': 'the y coordinate within the electrode group', 'default': None},
+    {'name': 'rel_z', 'type': 'float', 'doc': 'the z coordinate within the electrode group', 'default': None},
+    {'name': 'reference', 'type': str, 'doc': 'Description of the reference electrode and/or reference scheme\
+        used for this  electrode, e.g.,"stainless steel skull screw" or "online common average referencing". ',
+        'default': None},
+    {'name': 'enforce_unique_id', 'type': bool, 'doc': 'enforce that the id in the table must be unique',
+     'default': True}
+)
+
+
+class ElectrodeTable(DynamicTable):
```

Comment:
I'm wondering whether it would make sense to define the ElectrodeTable in ``ecphys.py`` instead and import it in ``file.py``